tests/time_tests/src/timetests_helper/statistics_writer.h

```diff
@@ -46,12 +51,40 @@ class StatisticsWriter {
     }
   }
 
+  /**
+   * @brief Compute order for statistics operations.
+   */
+  void addOrderCount(const std::string name) {
+    order++;
+    order_structure.insert(std::make_pair(name, order));
```

Comment:
can `order` be eliminated by index of record in map? Like described here - https://stackoverflow.com/questions/17863079/get-index-of-element-in-c-map

Reply:
So `order_structure` can be converted to vector, and it's simple to get index of it